test/generate-testcase.js

```diff
@@ -75,17 +75,21 @@ function fetchSource(url, callbackFn) {
       if (debug) {
         console.log("End received");
       }
-      // Sanitize:
-      htmltidy(serializeDocument(jsdom(rv)), {
-        "indent": true,
-        "indent-spaces": 4,
-        "output-xhtml": true,
-        "wrap": 0
-      }, callbackFn);
+      sanitizeSource(rv, callbackFn);
     });
   });
 }
 
+function sanitizeSource(html, callbackFn) {
+  htmltidy(serializeDocument(jsdom(html)), {
+    "indent": true,
+    "indent-spaces": 4,
+    "numeric-entities": true,
+    "output-xhtml": true,
+    "wrap": 0
```

Comment:
Nitpick: can you add a trailing comma after the last property as well? This makes it easier to add/remove entries in objects/arrays.